resources/analysisTools/roddyTools/lib.sh

```diff
@@ -4,6 +4,7 @@
 # Distributed under the MIT License (license terms are at https://github.com/DKFZ-ODCF/AlignmentAndQCWorkflows).
 #
 
+DEFAULTPLUGIN_LIB___ERREXIT=$(if [[ $SHELLOPTS =~ "errexit" ]]; then echo "errexit"; fi)
```

Comment:
There was a problem with storing and restoring the 'errexit' option with the Bash version used in the test.